tf/k8s.tf

```diff
@@ -60,4 +60,13 @@ resource "azurerm_kubernetes_cluster" "k8s" {
     role_based_access_control {
         enabled = true
     }
-}
\ No newline at end of file
+}
+
+resource "azurerm_sql_server" "sql" {
+  name                         = "mics-pegasus"
+  resource_group_name          = "${azurerm_resource_group.rg.name}"
+  location                     = "West US"
+  version                      = "12.0"
+  administrator_login          = "4dm1n157r470r"
```

Comment:
Lol, I think we have to use actual credentials.  I'm going to plumb these through the secrets pipeline.